lib/socrates/core/dispatcher.rb

```diff
@@ -151,8 +152,10 @@ def done_transitioning?(state)
       end
 
       def handle_action_error(error, session, state)
-        @logger.warn "Error while processing action #{state.data.state_id}/#{state.data.state_action}: #{error.message}"
-        @logger.warn error
+        @error_logger.error "Error while processing action \
+          #{state.data.state_id}/#{state.data.state_action}: #{error.message}"
+        @error_logger.error "For user: #{session.user}"
```

Comment:
Let's leave user out for now. We have another story for addressing this and it's a little complicated: user may be a big bag/hash of information, including stuff that's not as relevant.

Reply:
OK.